walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,104 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.lang.Math;
 
 public final class FindMeetingQuery {
+
+  private ArrayList<TimeRange> meetings = new ArrayList<TimeRange>();
```

Comment:
You should use List for specifying the type instead of ArrayList, so it will be 
```
private List<TimeRange> meetings = new ArrayList<TimeRange>();
```

The principle is you should code to interfaces, not implementations. List is an interface and ArrayList is an implementation of the List interface. By coding to interfaces, in the future you can easily swap it out for another List implementation (e.g. LinkedList). 

Some resources are linked here: https://stackoverflow.com/questions/383947/what-does-it-mean-to-program-to-an-interface